src/test/javascript/spec/component/shared/button.component.spec.ts

```diff
@@ -3,32 +3,32 @@ import { By } from '@angular/platform-browser';
 import { TranslateModule } from '@ngx-translate/core';
 import { JhiLanguageHelper } from 'app/core/language/language.helper';
 import { DebugElement } from '@angular/core';
-import { SinonSpy, spy } from 'sinon';
-import * as chai from 'chai';
-import sinonChai from 'sinon-chai';
 import { ArtemisTestModule } from '../../test.module';
-import { ArtemisSharedComponentModule } from 'app/shared/components/shared-component.module';
 import { LocalStorageService, SessionStorageService } from 'ngx-webstorage';
 import { MockSyncStorage } from '../../helpers/mocks/service/mock-sync-storage.service';
 import { FeatureToggleService } from 'app/shared/feature-toggle/feature-toggle.service';
 import { MockFeatureToggleService } from '../../helpers/mocks/service/mock-feature-toggle.service';
 import { ButtonComponent } from 'app/shared/components/button.component';
-
-chai.use(sinonChai);
-const expect = chai.expect;
+import { MockProvider, MockDirective } from 'ng-mocks';
+import { empty } from 'rxjs';
+import { NgbTooltip } from '@ng-bootstrap/ng-bootstrap';
+import { FeatureToggleDirective } from 'app/shared/feature-toggle/feature-toggle.directive';
+import { TranslatePipeMock } from '../../helpers/mocks/service/mock-translate.service';
+import { TranslateDirective } from 'app/shared/language/translate.directive';
 
 describe('ButtonComponent', () => {
     let comp: ButtonComponent;
     let fixture: ComponentFixture<ButtonComponent>;
     let debugElement: DebugElement;
 
-    let clickSpy: SinonSpy;
+    let clickSpy: jest.SpyInstance;
 
     beforeEach(async () => {
         return TestBed.configureTestingModule({
-            imports: [TranslateModule.forRoot(), ArtemisTestModule, ArtemisSharedComponentModule],
+            imports: [TranslateModule.forRoot(), ArtemisTestModule],
```

Comment:
You can remove the `TranslateModule.forRoot()`, I just checked it, tests run without it as well ;) 
```suggestion
            imports: [ArtemisTestModule],
```